quarkus/extensions/pom.xml

```diff
@@ -79,7 +103,6 @@
             <plugin>
                 <groupId>org.jboss.jandex</groupId>
                 <artifactId>jandex-maven-plugin</artifactId>
```

Comment:
I noticed beans.xml earlier, I think we should use a consistent approach shouldn't we? Either use beans.xml or use jandex plugin?

Reply:
I'm not sure if this declaration is making any difference. But yeah, we should have a consistent approach and we basically need beans.xml to make Hibernate work.